sdk/cxx/include/sawtooth/global_state.h

```diff
@@ -40,31 +40,31 @@ class GlobalState {
     // multiple values, it is encouraged to use the batch get overload defined
     // below. The function returns false if the value is not retrieved, true if
     // it is found in the GlobalStore, and throws an exception if an error occurs
-    bool Get(std::string* out_value, const std::string& address) const;
+    bool get_state(std::string* out_value, const std::string& address) const;
```

Comment:
The casing should remain the same for this library:  GetState, SetState, DeleteState